Description: fix FTBFS when building the java bindings
 I'm not yet entirely sure of why, but all files tends to go inside that branch,
 despite all ${java_file} matching "${OpenCV_BINARY_DIR}/".  I believe it has
 something to do with the content of ${OpenCV_BINARY_DIR}, in particular the
 presence of '+' and/or '.' in the build path.
 For now let's just patch it out until something better comes around.
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: not-needed
Last-Update: 2017-10-27

--- a/modules/java/CMakeLists.txt
+++ b/modules/java/CMakeLists.txt
@@ -291,8 +291,6 @@
   endif()
   if(__configure)
     configure_file("${java_file}" "${java_src_dir}/${output_name}" @ONLY)
-  elseif(NOT "${java_file}" MATCHES "${OpenCV_BINARY_DIR}/")
-    configure_file("${java_file}" "${java_src_dir}/${output_name}" COPYONLY)
   else()
     add_custom_command(OUTPUT "${java_src_dir}/${output_name}"
                        COMMAND ${CMAKE_COMMAND} -E copy_if_different "${java_file}" "${java_src_dir}/${output_name}"
